health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/GeoPode.js

```
@@ -19,112 +19,160 @@ const GeoPode = () => {
     const defaultHierarchyType = searchParams.get("defaultHierarchyType")
     const newHierarchy = JSON.parse(searchParams.get("newHierarchy"));
     const [showFinalPopup, setShowFinalPopup] = useState(false);
-
+    let locale = Digit?.SessionStorage.get("initData")?.selectedLanguage || "en_IN";
```

Biome

[error] This let declares a variable that is only assigned once.

'locale' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)